app/controllers/hearing_days_controller.rb

```diff
@@ -0,0 +1,67 @@
+class HearingDaysController < ApplicationController
+  before_action :load_and_authorize_application, :load_hearing_days, :load_hearing_details,
+                :load_hearing_events
+
+  add_breadcrumb :search_filter_breadcrumb_name, :new_search_filter_path
+  add_breadcrumb :search_breadcrumb_name, :search_breadcrumb_path
+
+  def show
+    add_breadcrumb @application.application_title
+    add_breadcrumb t(".breadcrumb", day: @hearing_day.day_string)
+  end
+
+  private
+
+  def load_and_authorize_application
+    @application = CourtDataAdaptor::Query::CourtApplication.new(params[:court_application_id]).call
+    authorize! :show, @application
+  rescue JsonApiClient::Errors::ServiceUnavailable => e
+    Sentry.capture_exception(e)
+    redirect_to controller: :errors, action: :internal_error
+  rescue JsonApiClient::Errors::NotFound
+    redirect_to controller: :errors, action: :not_found
+  end
+
+  def load_hearing_days
+    load_hearing_day
+    load_adjacent_hearing_days
+  end
+
+  def load_hearing_details
+    details = CdApi::Hearing.find(@hearing_day.hearing.id, params: { date: @hearing_day.date })
+    @hearing_details = helpers.decorate(details, CdApi::HearingDecorator)
+    @hearing_details.current_sitting_day = @hearing_day.sitting_day
+    @hearing_details.skip_mapping_counsels_to_defendants = true
```

Comment:
What is the purpose of this flag? Skip the mapping as it isn't needed?

Reply:
So the logic that works out which defence counsels attended on a given day also tries to work out which defendants were being represented by which counsel (so it can say things like "Barry McBarristerface (for Deffy McDefendant)" in the UI. The logic is both broken and redundant for appeal hearings (broken because it specifically tries to pull defendants from a prosecution case object that isn't available, and redundant because appeals don't concern themselves with multiple defendants), but rather than re-implement the "which counsels were there on which day" logic, I wanted to reuse the existing logic, simply skipping over the broken/redundant stuff, so that we end up with "Barry McBarristerface" and no parentheses.

Reply:
Okay, got it. Thank you!